ci-operator/step-registry/baremetalds/packet/check/baremetalds-packet-check-commands.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+echo "************ baremetalds env setup ************"
+env | sort
+
+set +x
+
+PACKET_PROJECT_ID=$(cat ${CLUSTER_PROFILE_DIR}/.packet-kni-vars|grep packet_project_id|awk '{print $2}')
+PACKET_AUTH_TOKEN=$(cat ${CLUSTER_PROFILE_DIR}/.packet-kni-vars|grep packet_auth_token|awk '{print $2}')
+SLACK_AUTH_TOKEN=$(cat ${CLUSTER_PROFILE_DIR}/.slackhook)
+
+# Initial check
+if [ "${CLUSTER_TYPE}" != "packet" ] ; then
+    echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"
+    exit 1
+fi
+
+#Packet API call to get list of servers in project
+servers="$(curl -X GET --header 'Accept: application/json' --header "X-Auth-Token: ${PACKET_AUTH_TOKEN}"\
+ "https://api.packet.net/projects/${PACKET_PROJECT_ID}/devices?exclude=root_password,ssh_keys,created_by,project,project_lite\
+,ip_addresses,plan,meta,operating_system,facility,network_ports&per_page=1000")"
+
+#Assuming all servers created more than 4 hours = 14400 sec ago are leaks
+leaks="$(echo "$servers" | jq -r '.devices[]|select((now-(.created_at|fromdate))>14400)')"
```

Comment:
I know @andfasano added some tags/labels to our hosts -- is there a way to differentiate jobs from that come of a CI job vs. one that comes from @cluster-bot? The latter last 12 hours I think.

We can solve it later I suppose, 4 hours is more than enough for now

Reply:
(We don't have cluster-bot setup yet for e2e-metal-ipi, but hopefully soon)

Reply:
Tags were temporary removed after moving to ansible and new packet project. That's my next task. Need to add labels handling to packet ansible module, then re-introduce them here.